books/free-programming-books-zh.md

```diff
@@ -601,7 +592,7 @@
 
 ### Lua
 
-* [Lua 5.3 参考手册](http://www.w3cschool.cc/manual/lua53doc/contents.html)
+* [Lua 5.3 参考手册](https://www.runoob.com/lua/)
```

Comment:
The manual is linked as the last item of the menu (at the bottom)

![image](https://user-images.githubusercontent.com/3125580/183254561-1fefc8de-11c3-43c0-99b3-7eb319e1414c.png)


```suggestion
* [Lua 5.3 参考手册](https://www.runoob.com/manual/lua53doc/)
```

Reply:
for this I used `.../lua/`, they will have access to both resources

Reply:
The problem is that `.../lua/` is more like an interactive tutorial, meanwhile `.../manual/lua53doc/` is the analogous resource than doesn't work currently. I'd prefer the second